scripts/install-mender.sh

```diff
@@ -210,7 +210,7 @@ add_repo() {
         exit 1
     fi
     if test -f /etc/apt/sources.list.d/mender.list && \
```

Comment:
```
MENDER_APT_SOURCE="/etc/apt/sources.list.d/mender.list"
```
wouldn't that be nice?

as a review I actually double checked that `"/etc/apt/sources.list.d/mender.list"` is actually created by the get.medner.io script.

Reply:
okey